openmdao/drivers/genetic_algorithm_driver.py

```diff
@@ -807,11 +830,55 @@ def encode(self, x, vlb, vub, bits):
         Returns
         -------
         ndarray
-            Population of points, encoded.
+            Single population member, encoded.
         """
         interval = (vub - vlb) / (2**bits - 1)
         x = np.maximum(x, vlb)
         x = np.minimum(x, vub)
         x = np.round((x - vlb) / interval).astype(np.int)
         byte_str = [("0" * b + bin(i)[2:])[-b:] for i, b in zip(x, bits)]
-        return np.array([int(c) for s in byte_str for c in s])
+        result = np.array([int(c) for s in byte_str for c in s])
+        if self.gray_code:
+            result = self.to_gray(result)
+        return result
+
+    def to_gray(self, g):
+        """
+        Convert a Gray coded binary array representing a single population member
+        to normal binary coding.
+
```

Comment:
Docstring descriptions seem tot be reversed. This one is to_gray, but the string says convert from gray.

Reply:
fixed